Userland/Libraries/LibGfx/ImageFormats/ISOBMFF/Enums.h

```diff
@@ -20,6 +20,9 @@ namespace Gfx::ISOBMFF {
     ENUMERATE_ONE(JPEG2000ChannelDefinitionBox, "cdef")        \
     ENUMERATE_ONE(JPEG2000ComponentMappingBox, "cmap")         \
     ENUMERATE_ONE(JPEG2000ColorSpecificationBox, "colr")       \
+    ENUMERATE_ONE(JPEGXLCodestreamBox, "jxlc")                 \
+    ENUMERATE_ONE(JPEGXLLevelBox, "jxll")                      \
+    ENUMERATE_ONE(JPEGXLSignatureBox, "JXL ")                  \
```

Comment:
more files to add these to are mentioned in https://github.com/SerenityOS/serenity/pull/25762, https://github.com/SerenityOS/serenity/pull/25766, https://github.com/SerenityOS/serenity/pull/25671, https://github.com/SerenityOS/serenity/pull/24016

Reply:
It seems to already be all good, these files use the same extension as regular JPEG XL files:
![image](https://github.com/user-attachments/assets/a9bd0498-dfda-46aa-b32c-5c89bef5f5c3)

Reply:
Derp, I thought this updated the file extension enum, but it's a box handling enum. Apologies.

Reply:
No problem :)